xmodule/js/src/capa/display.js

```diff
@@ -696,7 +701,7 @@
         };
 
         Problem.prototype.reset = function() {
-            return this.disableAllButtonsWhileRunning(this.reset_internal, false);
+            return this.disableAllButtonsWhileRunning(this.reset_internal, false, resetClick);
```

Comment:
Is it possible to remove the current state instead of disabling the submit button?